javascript/src/test/pages/Home/Home.test.js

```diff
@@ -1,9 +1,25 @@
 import React from "react";
 import { render } from "@testing-library/react";
 import Home from "main/pages/Home/Home";
+import useSWR from "swr";
+jest.mock("swr");
 
 describe("Home tests", () => {
+  const filterData = [
+    {id: "1",
+  activeQuarter:"something that exists that isnt all"}
```

Comment:
formatting could be a little bit better here
```suggestion
    {
      id: "1",
      activeQuarter:"something that exists that isnt all"
    }
```